src/main/java/com/sysdig/jenkins/plugins/sysdig/NewEngineReportConverter.java

```diff
@@ -331,99 +331,92 @@ private String getRuleString(JSONArray rule) {
         case "denyCVE":
           break;
         case "vulnSeverity":
-          ruleResult.add(" Severity is " + extra.getString("level"));
+          ruleResult.add("Severity greater than or equal " + extra.getString("level"));
+          break;
+        case "vulnSeverityEquals":
+          ruleResult.add("Severity equal " + extra.getString("level"));
           break;
         case "vulnIsFixable":
-          ruleResult.add(" Fixable");
+          ruleResult.add("Fixable");
           break;
         case "vulnIsFixableWithAge":
           int days = extra.getInt("age");
           String period = " days";
-          if (days < 2) {
-            period = " day";
-          }
-          ruleResult.add(" Fixable since " + days + period);
+          period = days < 2 ? " day" : " days";
```

Comment:
maybe you can remove the line above now

Reply:
unfortunately it's still needed as declaration of the variable for the other cases where it's being used